net/upnp/src/www/status_upnp.php

```diff
@@ -57,7 +57,7 @@
 <?php
           if (empty($config['installedpackages']['miniupnpd']['config'][0]['iface_array']) || empty($config['installedpackages']['miniupnpd']['config'][0]['enable'])): ?>
           <header class="content-box-head container-fluid">
-            <h3><?= gettext('UPnP is currently disabled.') ?></h3>
+            <h3><?= gettext('UPnP IGD & PCP/NAT-PMP is currently disabled.') ?></h3>
```

Comment:
```suggestion
            <h3><?= gettext('UPnP is currently disabled.') ?></h3>
```

Again here this is not a configuration workflow. I'm ok with the rest then.